src/types.ts

```diff
@@ -176,7 +184,19 @@ export type Meeting = [
    * an integer index into `caches.dateRanges`,
    * specifying the start/end date of the class this semester
    */
-  dateRangeIndex: number
+  dateRangeIndex: number,
+  /**
+   * integer index into caches.finalDates,
+   * specifying the date at which the final is
+   * -1 when no match could be found
+   */
+  finalDateIndex: number,
+  /**
+   * integer index into caches.finalTimes,
+   * specifying the time at which the final is
+   * -1 when no match could be found
+   */
+  finalTimeIdx: number
```

Comment:
Grabbing latest docs from https://github.com/gt-scheduler/website/pull/134 to keep them in sync.

```suggestion
  /**
   * integer index into caches.finalDates,
   * specifying the date at which the final is
   * -1 when no match could be found and
   * as a default value
   */
  finalDateIndex: number,
  /**
   * integer index into caches.finalTimes,
   * specifying the time at which the final is
   * -1 when no match could be found
   * and as a default value
   */
  finalTimeIdx: number
```